nodes/config/ui_base.js

```diff
@@ -126,6 +126,23 @@ module.exports = function (RED) {
 
             uiShared.app.use(config.path, uiShared.httpMiddleware, express.static(path.join(__dirname, '../../dist')))
 
+            // Redirect any requests with a trailing slash to the same path without the trailing slash
+            uiShared.app.use((req, res, next) => {
+                // Skip if not a GET or HEAD request
+                if (!(req.method === 'GET' || req.method === 'HEAD')) {
+                    next()
+                    return
+                }
+
+                if (req.path.slice(-1) === '/' && req.path.length > 1) {
+                    const query = req.url.slice(req.path.length)
+                    const safePath = req.path.slice(0, -1).replace(/\/+/g, '/')
+                    res.redirect(301, safePath + query)
+                } else {
+                    next()
+                }
```

Comment:
I suggest to add a comment here mentioning the underlying issue that this fixes as it's not easy to remember the logic that requires this :)

Reply:
Agree with adding a comment here, otherwise, this looks good to go

Reply:
@Steve-Mcl can you action this please? Then we can get a new Dashboard release out.

Reply:
Done Joe.